src/components/Map/Map.tsx

```diff
@@ -24,7 +24,7 @@ export default function Map({ countries, disputedAreas }: MapProps) {
       style={{ height: '100%', width: '100%', zIndex: 1 }}
     >
       <AlertContainer />
-      {countries && <VectorTileLayer countries={countries} disputedAreas={disputedAreas} />}
+      {countries && <VectorTileLayer countries={countries} disputedAreas={disputedAreas} ipcData={ipcData} />}
```

Comment:
maybe a new layer would be needed for the normal VectorTile layer instead? @Tschonti

Reply:
I tried not to have additional layer, since having additional layer will have some sort of unnatural behaviour when zooming in or out (not sure if it's solvable or not)

Reply:
I think this will depend on **IPC/CH Country View**  if this will show details about country ipc phases thus, its better to have a seprate layer.